sites/platform/src/create-apps/app-reference/composable-image.md

```diff
@@ -598,6 +598,61 @@ much like you would plug a hard drive into your computer to transfer data.
 
 {{% /note %}}
 
+### Mount types
+
+Each mount type has a specific use case and understanding how to use each type effectively will help optimize your application.
+
+#### `storage` mount
+
+The `storage` mount is a shared network storage that is automatically added by the platform. It is required for applications that need to share files across multiple instances—such as user-uploaded media or shared assets. You can configure the size of this mount via the {{% vendor/name %}} UI.
+
+Pros:
+- Shared across all app and worker instances.
```

Comment:
The worker instances do not get to share the same mount as the app instances for this kind of storage. You need to declare an explicit network-storage service and use that if you want to share a mount between app and workers. The default `storage` mount is not shareable. The purpose is for horizontally-scaled applications to use the same files.